src/components/Listing/Listing.tsx

```diff
@@ -37,7 +37,7 @@ export const Listing = ({
   className,
   variant = 'link_list'
```

Comment:
I was wondering if we need this to be default, but I guess when we create a custom page and want to add a list of links, it could be easier to just add `<Listing>` without any props for the correct markup to be implemented.